src/main/src/main/java/org/geoserver/security/GeoServerSecurityManager.java

```diff
@@ -608,9 +610,18 @@ public AuthenticationCache getAuthenticationCache() {
         return authCache;
     }
 
-    AuthenticationCache lookupAuthenticationCache() {
+    private AuthenticationCache lookupAuthenticationCache() {
```

Comment:
I can't say I remember why I did that. It isn't needed by the patch so I'll remove it just to be safe.  Thanks.